diff -Nru make-3.78.1/make.texinfo make-3.78.1.new/make.texinfo
--- make-3.78.1/make.texinfo	Thu Sep 23 19:40:19 1999
+++ make-3.78.1.new/make.texinfo	Thu Oct  7 03:30:35 1999
@@ -24,9 +24,9 @@
 @c Combine the program and concept indices:
 @syncodeindex pg cp
 
-@dircategory GNU Packages
+@dircategory  Programming tools:
 @direntry
-* Make: (make).            Remake files automatically.
+* Make: (make).				The GNU make utility
 @end direntry
 
 @ifinfo
