--- make-3.79.1/configure.in.jj	Fri Jun 23 12:09:41 2000
+++ make-3.79.1/configure.in	Tue Aug  1 16:12:12 2000
@@ -75,7 +75,7 @@ AC_MSG_RESULT($ac_cv_check_symbol_$1)])d
 
 # Solaris 2.5.1 needs -lposix4 to get the clock_gettime function.
 # Solaris 7 prefers the library name -lrt to the obsolescent name -lposix4.
-AC_SEARCH_LIBS(clock_gettime, [rt posix4])
+AC_SEARCH_LIBS(clock_gettime, [posix4])
 if test "$ac_cv_search_clock_gettime" != no; then
   AC_DEFINE(HAVE_CLOCK_GETTIME, 1,
             [Define if you have the clock_gettime function.])
yy